src/routers/account.js

```diff
@@ -249,7 +254,7 @@ router.post('/login', async (request, response) => {
 
 	response.cookie('ph', encryptedBody.toString('hex'), { domain: '.pretendo.network' });
 
-	response.redirect('/account');
+	response.redirect(redirect || '/account');
```

Comment:
this seems unsafe, validate the redirect url please. (startswith / is probably sufficient)